### PR TITLE
Improve diff view

### LINKED ISSUE #695: Gap element wrongly positioned in diff widget
The gap element in the diff view is still added in the DOM but is not properly positioned between the 2 editors.

### PR DESCRIPTION
**Pull request by fcollonval**
Restore gap element in diff view
Improve alignment (homogeneous grid usage)
Remove unneeded gutter in diff view

image

---

Fixes #695

### CODE REVIEW

**Comment by fcollonval** [comment]
bot please update playwright snapshots

**Comment by fcollonval** [comment]
Kicking the CI

**Comment by fcollonval** [comment]
bot please update playwright snapshots

**Comment by fcollonval** [comment]
Kicking the CI

File: nbdime/webapp/templates/diff.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h3>Notebook Diff</h3>
         <button id="nbdime-close" type="checkbox" style="display: none">Close tool</button>
         <button id="nbdime-export" type="checkbox" style="display: none">Export diff</button>
       </div>
-      <div id=nbdime-header-banner>
+      <div id=nbdime-header-banner class="cm-merge-2pane">
```
Comment (fcollonval) [inline nbdime/webapp/templates/diff.html:25]:
```suggestion
      <div id="nbdime-header-banner" class="cm-merge-2pane">
```

File: nbdime/webapp/templates/difftool.html [hunk 2]
```diff
@@ -10,7 +10,7 @@ <h3>Notebook Diff</h3>
         <button id="nbdime-close" style="display: none">Close tool</button>
         <button id="nbdime-export" style="display: none">Export diff</button>
       </div>
-      <div id=nbdime-header-banner>
+      <div id=nbdime-header-banner class="cm-merge-2pane">
```
Comment (fcollonval) [inline nbdime/webapp/templates/difftool.html:13]:
```suggestion
      <div id="nbdime-header-banner" class="cm-merge-2pane">
```

**Review by krassowski — APPROVED** [review]
LGTM, thanks!

**Pull request merged**